gradle.properties

```diff
@@ -30,7 +30,7 @@ LINT_VERSION            = 26.3.2
 ROBOLECTRIC_VERSION     = 3.3.2
 COMMONS_IO_VERSION      = 2.6
 CONDUCTOR_VERSION       = 2.1.5
-KOMPILE_TESTING_VERSION = 0.1.1
+KOMPILE_TESTING_VERSION = 0.1.3
```

Comment:
Can you rebase?👀https://github.com/permissions-dispatcher/PermissionsDispatcher/pull/628